### ISSUE TITLE
Remove `addNodeStatusCheckForPGFallback()` in a future release

### ISSUE THREAD

**Issue by seledwards** [GENI-NSF/geni-tools#393]
In #389 introduced a fallback to read login info from SliverStatus for users not listed in the manifest.  

This should be removed in a later release of the code.  Or make a new flag (like `--noFallbackToStatusForPG`) to disable it by default and optionally allow people to enable it.

 _Imported from trac ticket [#393](http://trac.gpolab.bbn.com/gcf/ticket/393),  created by sedwards on 10-09-2013 at 10:25, last modified: 09-05-2014 at 20:38_
   CCing: gpo-expt-support@...

**Comment by seledwards** [comment]
Before removing this, beware that currently users added via a call to `poa` don't appear in the login info in the manifest.  So the only way to get their login info is via `SliverStatus`.

Consider not removing this until the manifest gets updated after a call to `poa`.

_Trac comment by sedwards (github user: seledwards) on 05-21-2014 at 13:59_

**Comment by seledwards** [comment]
This should actually be fixed in 2.7.  We should change the default of `--noFallbackToStatusForPG` to false so that any users using omni at an aggregate that isn't up-to-date can still use it if needed.

_Trac comment by sedwards (github user: seledwards) on 07-17-2014 at 14:06_

**Comment by seledwards** [comment]
Replaced `--noFallbackToStatusForPG` with `--fallbackToStatusForPG` which has the opposite meaning.

To test, run the equivalent of the following two commands:
```
readyToLogin.py slicename -a gatech-ig --fallbackToStatusForPG
readyToLogin.py slicename -a gatech-ig
```

The output for the first command should include the following phrase near the top:
```
--noFallbackToStatusForPG
```
The output of the second command will not.

_Trac comment by sedwards (github user: seledwards) on 09-05-2014 at 20:38_
